app/Http/Controllers/SubjectController.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\SubjectRequest;
+use App\Models\Course;
+use App\Models\CourseUser;
+use App\Models\Role;
+use App\Models\Subject;
+use App\Models\SubjectUser;
+use App\Models\Task;
+use Illuminate\Http\Request;
+
+class SubjectController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $courses = Course::all();
+        $subjects = Course::withCount('subjects')->get();
+        $users = Subject::withCount('subjectUsers')->get();
+
+        for ($i = 0; $i < count($courses); $i++) {
+            $listSubjects = Subject::where('course_id', $courses[$i]->id)
+                ->paginate(10);
+            $courses[$i]->subjects = $subjects[$i]->subjects_count;
+            $courses[$i]->users = $users[$i]->course_users_count;
+            $courses[$i]->listSubjects = $listSubjects;
+        }
```

Comment:
có thể dùng foreach cho gọn

Reply:
Đoạn này em dùng vòng for để lấy giá trị cùng 1 vị trí trong mảng $subjects ạ